### DISCUSSION TITLE
TOKEN_INVALID

### DISCUSSION THREAD

**Discussion by Chriss2422** [eritislami/evobot#1166] [Q&A]
it always says this even when i change the token and i have tried to put the token in the client.login(TOKEN); but it doesn work pls help me
![image](https://user-images.githubusercontent.com/102837979/169886630-53d45e2d-e331-4917-9712-1e7c43139084.png)

**Comment by kotru21** [comment]
Put your token in config json(in Token field)

**Comment by ZaibatsuX** [comment]
try resetting the token its working for me

**Comment by Rnbsov** [comment]
you should address to variable like `config.TOKEN`

**Comment by AakashRawat04** [comment]
what you can do is just make a new key in the config.json file and then use that field in your code. also, try to reset the token. i dont find any problem with it.
